src/pat/validation/validation.js

```diff
@@ -58,20 +58,21 @@ define([
             this.errors = 0;
             this.options = parser.parse(this.$el, opts);
             this.$inputs = this.$el.find(':input[name]');
-            this.$el.find(":input[type=number]").on('keyup mouseup', _.debounce(function (ev) {
+
+            // Fire - debounced - when value of input elements changes
+            // See: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event
+            this.$el.find("input:not([type=button]):not([type=reset]):not([type=submit]:not([type=checkbox]):not([type=radio])), select, textarea, *[contenteditable]:not([contenteditable=false])").on('input', _.debounce(function (ev) {
+                this.validateElement(ev.target);
+            }.bind(this), 500));
+            // See: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event
+            this.$el.find("input[type=checkbox]), input[type=radio]").on('change', _.debounce(function (ev) {
                 this.validateElement(ev.target);
             }.bind(this), 500));
-            this.$inputs.on('change.pat-validation', function (ev) { this.validateElement(ev.target); }.bind(this));
```

Comment:
I don't know yet how a ``change.pat-validation`` event is triggered...?
Same with ``submit.pat-validation`` and ``click.pat-validation``...?